### PR TITLE
[KIECLOUD-181]  Refactoring RHPAM quickstarter for JDK 11 compatibility

### PR DESCRIPTION
**Pull request by ricardozanini**
See:
https://issues.jboss.org/browse/KIECLOUD-181

Signed-off-by: Ricardo Zanini <zanini@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: quickstarts/library-process/library/pom.xml [hunk 1]
```diff
@@ -1,98 +1,194 @@
...
+	<modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.openshift.quickstarts</groupId>
-    <artifactId>rhpam-kieserver-library</artifactId>
-    <version>1.4.0-SNAPSHOT</version>
-    <packaging>jar</packaging>
+	<!-- Can't have a parent definition because of the way the OpenShift builds 
+		the kjar application. This project must be independent of its parent. -->
+	<groupId>org.openshift.quickstarts</groupId>
+	<version>1.5.0-SNAPSHOT</version>
```
Comment (spolti) [inline quickstarts/library-process/library/pom.xml]: you also need to update the application templates which uses this quickstart.

**Comment by sutaakar** [comment]
@ricardozanini Please use common IDE formatting configuration if possible - see https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration .

File: quickstarts/library-process/library-client/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/client/LibraryClient.java [hunk 2]
```diff
@@ -303,31 +303,38 @@ boolean returnLoan(Loan loan) {
...
-                            new X509TrustManager() {
-                                public void checkClientTrusted(X509Certificate[] chain, String authType) throws CertificateException {}
-                                public void checkServerTrusted(X509Certificate[] chain, String authType) throws CertificateException {}
-                                public X509Certificate[] getAcceptedIssuers() {
-                                    return null;
-                                }
-                            }
+                        context.init(null, new TrustManager[]{
+                                                              new X509TrustManager() {
+
```
Comment (errantepiphany) [inline quickstarts/library-process/library-client/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/client/LibraryClient.java]: Why introduce such a huge indent? Are you using spaces (preferred) or tabs?

File: quickstarts/library-process/library/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/Library.java [hunk 3]
```diff
@@ -17,15 +17,16 @@
...
-        new Object[] {"978-0-7360-9829-8", "Successful Sports Officiating", "American Sports Education Program.", 3},
-        new Object[] {"978-0-7434-8773-3", "The Time Machine", "H. G. Wells' story of a time traveler.", 8},
-        new Object[] {"978-1-101-15402-1", "The Island of Dr. Moreau", "H. G. Wells' story of what may be most relevant to modern ethical dimemmas.", 5},
-        new Object[] {"978-1-4000-5-80-2", "The Zombie Survival Guide", "Complete Protection from the Living Dead", 6},
-        new Object[] {"978-1-448-14153-1", "Doctor Who: Summer Falls", "Story of Amelia Williams.", 2},
-        new Object[] {"978-1-4516-7486-6", "Tesla, Man Out of Time", "Explores the brilliant and prescient mind of one of the twentieth century's greatest scientists and inventors, Nikola Tesla.", 4},
-        new Object[] {"978-1-59474-449-5", "Pride and Prejudice and Zombies", "The Classic Regency Romance -- Now with Ultraviolent Zombie Mayhem!", 7}
+    private static final Object[][] INIT = new Object[][]{
+                                                          new Object[]{"978-0-307-35193-7", "World War Z", "An Oral History of the Zombie War", 1},
```
Comment (errantepiphany) [inline quickstarts/library-process/library/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/Library.java]: Why introduce such a huge indent? Are you using spaces (preferred) or tabs?

**Comment by errantepiphany** [comment]
Oops sorry to be redundant. I juts saw Karel's comment about formatting...

File: quickstarts/library-process/library-client/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/client/LibraryClient.java (hunk 2 above)
Comment (ricardozanini) [reply quickstarts/library-process/library-client/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/client/LibraryClient.java]: I'm using spaces and this is a consequence of the droolsjbpm formatter profile. I can revert back to the original one, but will break the formatter profile.

File: quickstarts/library-process/library/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/Library.java (hunk 3 above)
Comment (ricardozanini) [reply quickstarts/library-process/library/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/Library.java]: See above.

File: quickstarts/library-process/library-client/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/client/LibraryClient.java (hunk 2 above)
Comment (errantepiphany) [reply quickstarts/library-process/library-client/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/client/LibraryClient.java]: Nah, it's fine. It's that big of a deal to me. :)

File: quickstarts/library-process/library/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/Library.java (hunk 3 above)
Comment (errantepiphany) [reply quickstarts/library-process/library/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/Library.java]: Same. :)

**Pull request merged**